_posts/2025-04-11-Hybrid-Search-Options-And-Practice.md

```diff
@@ -0,0 +1,295 @@
+---
+layout: post
+title:  "Building effective hybrid Search in OpenSearch: Techniques and best practices"
```

Comment:
Update title: 
"Building effective hybrid search in OpenSearch: Techniques and best practices"